Traceback (most recent call last):
  File "/Users/amirrezasokhankhosh/Documents/test/SSFL-BSFL/nodes/./node6.py", line 13, in <module>
    client = Client(port, ClientNN, malicious=False)
  File "/Users/amirrezasokhankhosh/Documents/test/SSFL-BSFL/split_fed/client.py", line 24, in __init__
    self.get_data()
  File "/Users/amirrezasokhankhosh/Documents/test/SSFL-BSFL/split_fed/client.py", line 29, in get_data
    training_dataset = datasets.FashionMNIST(
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/torchvision/datasets/mnist.py", line 103, in __init__
    raise RuntimeError("Dataset not found. You can use download=True to download it")
RuntimeError: Dataset not found. You can use download=True to download it
